README.md

```diff
@@ -65,7 +65,7 @@ Please file feature requests and bugs at the [issue tracker][tracker].
 
 [tracker]: https://github.com/dart-lang/linter/issues
 [lints]: http://dart-lang.github.io/linter/lints/
-[google-lints]: /example/google.yaml
+[package-pedantic]: https://github.com/dart-lang/pedantic

```

Comment:
I think it would be better to point to the` analysis_options.yaml` file directly.